### PR TITLE
chore(ui): Round of ui fixes

### LINKED ISSUE #514: Gradient in the side navigation is broken in dark mode.
The gradient looks hard coded for light mode. Should use standard level color styles.

image

### LINKED ISSUE #516: Change cursor to "not-allowed" on the "add org" button in the sidenav until user's handle is registered
When hovering over the add org "+" button, cursor should change to "not-allowed" until the user has registered their handle. Since the action is not possible from this button, the cursor should reflect that.

`cursor: not-allowed;` is the right cursor to use.

image

image

### PR DESCRIPTION
**Pull request by juliendonck**
- Fix sidebar gradient in dark mode: close #514 
- change cursor to not-allowed on hover of add org button -> closes #516

### CODE REVIEW

**Comment by juliendonck** [comment]
@rudolfs is this the same test failing as before or did I break something?

**Comment by rudolfs** [comment]
I skipped the one that was causing problems, but looks like there are more flaky tests : / Let's try rebuilding again.

**Comment by rudolfs** [comment]
We can avoid introducing another custom color by just using the existing palette and using a full rgba transparency (which isn't a color):

```diff
diff --git a/public/colors.css b/public/colors.css
index 0dc66a70..81b00eb8 100644
--- a/public/colors.css
+++ b/public/colors.css
@@ -31,7 +31,6 @@
   --color-foreground-level-4: #a9b9c7;
   --color-foreground-level-5: #90a0af;
   --color-foreground-level-6: #546474;
-  --color-sidebar-gradient: 235, 239, 243;
 }

 [data-theme="dark"] {
@@ -67,5 +66,4 @@
   --color-foreground-level-4: #5e6d7a;
   --color-foreground-level-5: #8594a1;
   --color-foreground-level-6: #d3dee8;
-  --color-sidebar-gradient: 41, 52, 61;
 }
diff --git a/ui/DesignSystem/Component/Sidebar.svelte b/ui/DesignSystem/Component/Sidebar.svelte
index b3e31b22..3c56a620 100644
--- a/ui/DesignSystem/Component/Sidebar.svelte
+++ b/ui/DesignSystem/Component/Sidebar.svelte
@@ -53,8 +53,8 @@
     left: 0;
     background: linear-gradient(
       0deg,
-      rgba(var(--color-sidebar-gradient), 1) 0%,
-      rgba(var(--color-sidebar-gradient), 0) 100%
+      var(--color-foreground-level-2) 0%,
+      rgba(0, 0, 0, 0) 100%
     );
   }
```

**Comment by rudolfs** [comment]
The register project menu item also should have the `not-allowed` icon in both the project listing as well as the project screen:

Screenshot 2020-06-19 at 17 01 46
Screenshot 2020-06-19 at 17 01 19

**Comment by juliendonck** [comment]
@rudolfs didn't think about the gradient stuff. makes sense!

File: ui/DesignSystem/Component/Sidebar.svelte [hunk 1]
```diff
@@ -53,8 +53,8 @@
     left: 0;
     background: linear-gradient(
       0deg,
-      rgba(var(--color-sidebar-gradient), 1) 0%,
-      rgba(var(--color-sidebar-gradient), 0) 100%
+      var(--color-foreground-level-2) 0%,
+      + rgba(0, 0, 0, 0) 100%
```
Comment (rudolfs) [inline ui/DesignSystem/Component/Sidebar.svelte:57]:
```suggestion
      rgba(0, 0, 0, 0) 100%
```

Comment (juliendonck) [reply ui/DesignSystem/Component/Sidebar.svelte]: 🤦

**Pull request merged**